nrfs/include/internal/services/nrfs_usb.h

```diff
@@ -32,6 +32,19 @@ typedef struct __NRFS_PACKED {
 	nrfs_ctx_t ctx; /**< Context of the message. */
 } nrfs_usb_disable_req_t;
 
+/** @brief USB LD Boolean On. */
+typedef struct __NRFS_PACKED {
+	nrfs_hdr_t hdr; /**< Header of the message. */
+	nrfs_ctx_t ctx; /**< Context of the message. */
+} nrfs_usb_ld_boolean_enable_req_t;
```

Comment:
Shouldn't name be aligned with the function, boolean was just temporary name, right?